### ISSUE TITLE
In my case, the app keeps telling me: "Access failed. Please update your app or contact support.". 

### ISSUE THREAD

**Issue by Davide3i** [krokofant/JodelXposed#68]
Since a couple of months I keep receiving this message when posting a new Jodel or voting:
"In my case, the app keeps telling me: "Access failed. Please update your app or contact support."

My Android version is 4.4.4.
I think it may be probably related to an old Android version, but I'm not sure.

**Comment by Unbrick** [comment]
Which Jodel and which JodelXposed version are you on?

**Comment by Davide3i** [comment]
Last Jodel version, but I'm having this problem since a couple of months.
My Xposed version is the last one supported for Android 4.4.4: v2.6.1.

I've compiled the last Jodel Xposed version from your repository.

**Comment by Unbrick** [comment]
The latest jodel version depends on the device. 

Did you try to clear your app-data?

**Comment by Davide3i** [comment]
Jodel version: 4.42.4.
Yep, I've tried to clear Jodel's app data and cache with no avail.
May it be caused by my old Android version?

**Comment by Unbrick** [comment]
Possibly. It could also be that you are banned. Can you provide a log?

**Comment by Davide3i** [comment]
Nope, I'm not banned: I'm able to use Jodel flawlessly without Jodel Xposed.

I'm attaching my log: hope it's enough. It contains both the Jodel log and infos about my device.

[Jodel.zip](https://github.com/krokofant/JodelXposed/files/962735/Jodel.zip)
